polaris-service/src/test/java/io/polaris/service/PolarisApplicationIntegrationTest.java

```diff
@@ -166,16 +166,16 @@ public static void setup(
 
   @AfterAll
   public static void deletePrincipalRole() {
-    try (Response deletePrResponse =
-        EXT.client()
-            .target(
-                String.format(
-                    "http://localhost:%d/api/management/v1/principal-roles/%s",
-                    EXT.getLocalPort(), PRINCIPAL_ROLE_NAME))
-            .request("application/json")
-            .header("Authorization", "Bearer " + userToken)
-            .header(REALM_PROPERTY_KEY, realm)
-            .delete()) {}
+    EXT.client()
```

Comment:
I believe it's intended to close deletePrResponse after the call, so you may need to keep it.

Reply:
There is a new "close" call at the bottom, does that capture the "Try" logic as well? I'm not familiar enough with the API to know

Reply:
I think it will, unless it crashes halfway. This is connected to my other few comments

Reply:
That sounds like it doesn't match the Try logic then :)

Reply:
if it crashes halfway, before the close() can be alled, then, in try-finally version it would crash before the try's resource is allocated, so there would be no close in that case either